tests/e2e/test_e2e.py

```diff
@@ -208,3 +211,19 @@ def test_prompt_manager_no_model_params_undeployed():
 
     with mgr.exec() as prompt:
         assert prompt.params is None
+
+
+def test_tracer_received_sigint_or_sigterm_cleanup():
```

Comment:
> test_tracer_received_sigint_or_sigterm_cleanup

it looks like this test is only testing SIGINT. can you add another one for SIGTERM? if you want you could use `pytest.parametrize `: https://docs.pytest.org/en/7.1.x/how-to/parametrize.html